mixins/localize-mixin.js

```diff
@@ -124,6 +124,9 @@ export const LocalizeMixin = dedupeMixin(superclass => class extends superclass
 	_generatePossibleLanguages() {
 		const langs = new Set();
 
+		langs.add('en');
+		langs.add('en-us');
```

Comment:
Any thoughts on if we still need `en-us`?

I don't think web components typically provide a US specific lang term for EN.

Reply:
They definitely shouldn't. If someone feels confident enough in their search skills to determine if any en-us files exist, I'd be happy to remove it if it's not necessary.